environments/icds-cas/app-processes.yml

```diff
@@ -189,6 +189,7 @@ celery_processes:
       num_workers: 6
     icds_dashboard_reports_queue:
       concurrency: 2
+  'celery112':
```

Comment:
`112` - is that intentional?

Reply:
Yes.

Reply:
It is done to avoid any conflict with existing machine , although in this case celery12 was already out of cluster but going forward I am planning to proceed like this only.